en/mkdocs.yml

```diff
@@ -106,6 +106,11 @@ nav:
                       - Adding an API State Change Workflow: learn/design-api/advanced-topics/adding-an-api-state-change-workflow.md
                       - Block Subscription to an API: learn/design-api/advanced-topics/block-subscription-to-an-api.md
           - Consume APIs:
+                - Account Management for Developer Portal:
+                      - Recover Password : learn/consume-api/account-management-dev-portal/recover-dev-portal-password.md
+                      - Change Password : learn/consume-api/account-management-dev-portal/change-dev-portal-password.md
+                      - Advanced Topics:
+                            - Adding a User Signup Workflow: learn/consume-api/account-management-dev-portal/advanced-topics/adding-a-user-signup-workflow.md
```

Comment:
User sign up workflow should move to where user-sign up is configured. develop->customizations perhaps?

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/1555/commits/dcc2e157314cf300788569c01c2ad89f0877c4b3